### PR TITLE
Adds docs for database driver

### PR DESCRIPTION
**Pull request by michaelfarrell76**
Show the different supported drivers in terraform docs for database integration.

@dmattia do these docs deploy automatically, or do i need to cut a release?

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,3 +1,4 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```
Comment (michaelfarrell76) [inline go.sum:1]: i think im doing something wrong..... 

was trying to debug this https://github.com/transcend-io/terraform-provider-transcend/actions/runs/3342880564/jobs/5535551097

Comment (dmattia) [reply go.sum:1]: This changed from the `go generate ./...` command?

**Review by dmattia — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (michaelfarrell76) [reply go.sum:1]: it required me to install something then i ran `go generate` no other args

**Comment by dmattia** [comment]
@michaelfarrell76 You'll need to cut a release after it merges (same way as for the sombra module you did). You'll also want to update all the versions in this repo like I did last in: https://github.com/transcend-io/terraform-provider-transcend/commit/1318e56917272eba8fdad02e911fc326f8d0ee09

**Comment by michaelfarrell76** [comment]
@dmattia updated version. but should i convert that go file change? maybe my go versions are out of whack

**Comment by dmattia** [comment]
@michaelfarrell76 Yeah I probably would. This repo does have a devcontainer btw to make sure your version is correct in the future

**Comment by michaelfarrell76** [comment]
@dmattia any idea why these tests are failing on a doc update?

**Comment by michaelfarrell76** [comment]
dev container was super easy to use until i forgot how to push a commit without the `gamp` alias 😆

**Pull request merged**